backend/api/Controllers/HealthController.cs

```diff
@@ -25,10 +24,8 @@ public class HealthController : ControllerBase
         /// Creates a new instances of a HealthController class, initializes it with the specified arguments.
         /// </summary>
         /// <param name="environment"></param>
-        /// <param name="logger"></param>
-        public HealthController(IWebHostEnvironment environment, ILogger<HealthController> logger)
```

Comment:
I'm assuming this is unrelated. What is the reason for this change?

Reply:
It's related in the sense this is the controller for the endpoint for the version information.  I've cleaned up the code to remove unused variables.